class.jetpack-admin.php

```diff
@@ -51,6 +52,39 @@ private function __construct() {
 
 		// Add module bulk actions handler
 		add_action( 'jetpack_unrecognized_action', array( $this, 'handle_unrecognized_action' ) );
+
+		// If the site has Jetpack Anti-Spam, change the Akismet menu label accordingly.
+		$site_products = Jetpack_Site_Products::get();
+		if ( false !== array_search( 'jetpack_anti_spam', array_column( $site_products, 'product_slug' ), true ) ) {
+			// Prevent Akismet from adding a menu item.
+			add_action(
+				'init',
+				function () {
+					remove_action( 'admin_menu', array( 'Akismet_Admin', 'admin_menu' ), 5 );
+				},
+				99
+			);
+			// Add an Anti-spam menu item for Jetpack.
+			add_action(
+				'jetpack_admin_menu',
+				function () {
+					add_submenu_page( 'jetpack', __( 'Anti-Spam', 'jetpack' ), __( 'Anti-Spam', 'jetpack' ), 'manage_options', 'akismet-key-config', array( 'Akismet_Admin', 'display_page' ) );
+				}
+			);
+			add_action( 'admin_enqueue_scripts', array( $this, 'akismet_logo_replacement_styles' ) );
+		}
+	}
+
+	/**
+	 * Generate styles to replace Akismet logo for the Jetpack logo. It's a workaround until we create a proper settings page for
+	 * Jetpack Anti-Spam. Without this, we would have to change the logo from Akismet codebase and we want to avoid that.
+	 */
+	private static function akismet_logo_replacement_styles() {
```

Comment:
This should not be static, as that breaks the callback on line 74. Static callables take a different format, see [the docs here](https://www.php.net/manual/en/language.types.callable.php).
```suggestion
	private function akismet_logo_replacement_styles() {
```

Reply:
TIL! I have updated the PR.

Reply:
Nevermind. `private` without `static` didn't work well. I'm getting this warning:
```
 PHP Warning:  call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method Jetpack_Admin::akismet_logo_replacement_styles() in /var/www/html/wp-includes/class-wp-hook.php on line 287
```

Wouldn't making the function just `static` work with something like:
```
add_action( 'admin_enqueue_scripts', array( $this, 'akismet_logo_replacement_styles' ) );
```

In the link you gave me, I see an example that looks like: 
```
// An example callback method
class MyClass {
    static function myCallbackMethod() {
        echo 'Hello World!';
    }
}
// Type 3: Object method call
$obj = new MyClass();
call_user_func(array($obj, 'myCallbackMethod'));
```

And from what I understand, `$this` should be an instance of the class, so it should work as the example (I believe).